setup.cfg

```diff
@@ -188,12 +188,13 @@ exclude =
     build,
     sunpy/data/sample.py,
     sunpy/extern/**,
+    sunpy/__init__.py,
```

Comment:
This might be overkill, but flake8 was testing that imports were at the top of the file.